ethportal-api/src/eth.rs

```diff
@@ -1,9 +1,16 @@
-use ethereum_types::U256;
+use ethereum_types::{H256, U256};
 use jsonrpsee::{core::RpcResult, proc_macros::rpc};
+use reth_rpc_types::Block;
 
 /// Web3 JSON-RPC endpoints
 #[rpc(client, server, namespace = "eth")]
 pub trait EthApi {
     #[method(name = "chainId")]
     async fn chain_id(&self) -> RpcResult<U256>;
+    #[method(name = "getBlockByHash")]
+    async fn get_block_by_hash(
+        &self,
+        block_hash: H256,
+        full_transactions: bool,
```

Comment:
Personally, I'm fine with `full_transactions`, but the [spec](https://ethereum.github.io/execution-apis/api-documentation/) does say `Hydrated Transactions` - I guess it uses `"hydrated"` as the field title.

Reply:
Hm, yeah I haven't seen a lot of consensus on that field name, different copies of the spec call it different things. I can't remember know how I got to "full," because I also remember considering hydrated... I don't feel strongly about it. I will switch it.